src/main/resources/minigame.json

```diff
@@ -0,0 +1,265 @@
+[
+    {
+        "type": "FILL",
+        "question": "Who is the owner of Camp Buddy?",
+        "answers": [
+            "Goro"
+        ],
+        "wrong": []
+    },
+    {
+        "type": "FILL",
+        "question": "What is the animal motif of Keitaro?",
+        "answers": [
+            "frog",
+            "frogs"
+        ],
+        "wrong": []
+    },
+    {
+        "type": "FILL",
+        "question": "What is the animal motif of Hiro?",
+        "answers": [
+            "raccoon",
+            "raccoons"
+        ],
+        "wrong": []
+    },
+    {
+        "type": "FILL",
+        "question": "What is the animal motif of Hunter?",
+        "answers": [
+            "bunny",
+            "bunnies"
+        ],
+        "wrong": []
+    },
+    {
+        "type": "FILL",
+        "question": "What is the animal motif of Natsumi?",
+        "answers": [
+            "beetle",
+            "beetles"
+        ],
+        "wrong": []
```

Comment:
If you ignore missing properties you don't have to have the wrong in there

Reply:
This doesn't work because when reading we have to initialize the field as something, or else we have to declare `Question.wrong` as a nullable field. It's simpler to have it be an empty array.